tests/test_io.py

```diff
@@ -44,8 +44,8 @@
 
 INVALID_OUT_FILES = [
     r"CARBONCYCLE.*OUT",
-    r".*SUBANN.*OUT",
-    r"DAT_VOLCANIC_RF\..*OUT",
+    r".*SUBANN.*\.BINOUT",
```

Comment:
Is this change to ensure that the .OUT files are still readable. I'm fairly sure that we exclude them in the actual reader.

Reply:
It's because we can read subannual `.OUT` files now. I deleted that restriction about annual timesteps in the ASCII readers (see https://github.com/openclimatedata/pymagicc/pull/250/files#diff-1e016ee4725860a3cd5d802fca30655bL194). The test to make sure this is all good is https://github.com/openclimatedata/pymagicc/pull/250/files#diff-cc4345db19ff44863e9122c74e9f383fR1170 (it checks that it's on a monthly grid and that annual times are still read as desired)